custom/icds/templates/icds/messaging/indicators/lus/missed_cf_visit_to_aww.txt

```diff
@@ -0,0 +1 @@
+AWC {{ awc }} hian complementary feeding report  hemi te tan hian a pe lo {{ beneficiary }}
```

Comment:
I suppose double spaces like ```report  hemi``` are acceptable?

Reply:
Remarkable catch. I suppose it's not a big deal here since that extra character _probably_ will not push this message into the two-SMS (based on length) category.